src/olympia/addons/serializers.py

```diff
@@ -275,11 +275,16 @@ def to_representation(self, obj):
             data['homepage'] = self.outgoingify(data['homepage'])
         if 'support_url' in data:
             data['support_url'] = self.outgoingify(data['support_url'])
-        if ('weekly_downloads' in data and
-                data.get('type') == amo.ADDON_TYPE_CHOICES_API[
-                    amo.ADDON_PERSONA]):
-            # weekly_downloads don't make sense for lightweight themes.
-            data.pop('weekly_downloads')
+        if data.get('type') == amo.ADDON_TYPE_CHOICES_API[amo.ADDON_PERSONA]:
```

Comment:
Doesn't really matter but why not just test for `obj.type == amo.ADDON_PERSONA` ?

Reply:
I... don't remember. Maybe I thought I didn't have access to `obj` somehow... I'll double-check and change if that works